official/mnist/mnist.py

```diff
@@ -71,22 +71,22 @@ def example_parser(serialized_example):
     label = tf.cast(features['label'], tf.int32)
     return image, tf.one_hot(label, 10)
 
-  dataset = tf.contrib.data.TFRecordDataset([filename])
+  dataset = tf.data.TFRecordDataset([filename])
 
+  # Parse each example in the dataset
+  dataset = dataset.map(example_parser)
```

Comment:
Let's make sure we make the changes described in https://github.com/tensorflow/tensorflow/blob/r1.4/tensorflow/contrib/data/README.md. Specifically `dataset.map` has changes for both `num_threads` and `output_buffer_size`.

Reply:
Thanks for finding this! Added dataset.prefetch (since num_threads/num_parallel_calls=1 is the default behavior, I've left it out)